doc/source/basics.rst

```diff
@@ -1788,6 +1797,36 @@ argument:
    s.sort_values()
    s.sort_values(na_position='first')
 
+.. _basics.sort_indexes_and_values:
+
+By Indexes and Values
+~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.22.0
+
+Strings passed as the ``by`` parameter to :meth:`DataFrame.sort_values` may
+refer to either columns or index levels.
```

Comment:
Same issue here with "index levels". Does changing it to "index names" make sense here, since that's what you're referring to?

Side-question, does this work on a regular (non-multiindex) index?

Reply:
I see you use "index level names" in the whatsnew. That seems like a good option, if you could use it here too.

Reply:
Changed to "index level names". 

And yes, this does work for both the `Index` and `MultIndex` cases.
@TomAugspurger